content/en/docs/spinnaker-user-guides/pacrd/_index.md

```diff
@@ -463,6 +463,29 @@ Events:
   Warning  PipelineValidationFailed  0s (x4 over 3s)        pipelines  artifact with id "a-nonsense-value" and name "" could not be found for this pipeline
 ```
 
+## Setting up New Relic
+
+If you want to setup a New Relic configuration you can add your license to your `patch.yaml` file like:
```

Comment:
If you want Armory to get data on how you use PaCRD and errors you encounter you can enable a New Relic configuration. Having this data helps Armory improve the product and provide better support. If you want further details on what data is sent and how it's used please connect with our team to talk to the product manager.

To enable this, you can add the New Relic license to the `patch.yaml` file as follows:

Reply:
Note to self: Perhaps I should make the data gathering into public and documented, let me think on that

Reply:
They can configure their own license and see the errors in their env, in case they want to send us metrics we need to provide them with our license.

Reply:
```suggestion
If you want to send Armory data about how you use PaCRD and the errors you encounter, you can enable a New Relic integration. Having this data helps Armory improve the product and provide better support. For information about what data is sent and how it is used, contact Armory.

Reply:
Ah interesting - in that case let me make some changes to the suggestions

Reply:
Tweaking Brian's suggestion:

If you want to monitor how PaCRD is used and the errors encountered, you can enable a New Relic integration. You can either send this data to your New Relic account or to Armory's New Relic account. If you choose to share the data with Armory, it helps us improve the product and provide better support. For information about what data is sent and how it is used, contact Armory, or simply enable it with your New Relic account first.

To enable this integration, add the New Relic license to the `patch.yaml` file as shown below. If you send it to Armory's New Relic account we will give you a license to use, otherwise use your New Relic account's license.